<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qomodoro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="toolsCombo">
          <item>
           <property name="text">
            <string>Tools...</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Preferences</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>About</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Quit</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="mainTabs">
        <property name="styleSheet">
         <string notr="true">QTabWidget::tab-bar {
            alignment: center;
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="timerTab">
         <attribute name="title">
          <string>Timer</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QStackedWidget" name="timerStacked">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="startPage">
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="2">
                 <widget class="QPushButton" name="startBtn">
                  <property name="text">
                   <string>Start</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="1">
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="0">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="4">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="timerPage">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QWidget" name="timerWidget" native="true">
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_9">
                   <item row="0" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <spacer name="horizontalSpacer_4">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QComboBox" name="changeState">
                         <item>
                          <property name="text">
                           <string>Pomodoro</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>Short Break</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>Long Break</string>
                          </property>
                         </item>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_5">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <spacer name="horizontalSpacer_6">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item alignment="Qt::AlignRight">
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>Cycles to the next long break:</string>
                         </property>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignLeft">
                        <widget class="QLabel" name="cyclesLeft">
                         <property name="text">
                          <string>TextLabel</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_7">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                      <widget class="QLabel" name="stopwatch">
                       <property name="font">
                        <font>
                         <pointsize>62</pointsize>
                        </font>
                       </property>
                       <property name="text">
                        <string>--:--</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <widget class="QPushButton" name="togglePause">
                         <property name="text">
                          <string>Pause</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="skip">
                         <property name="text">
                          <string>Skip</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="statsTab">
         <attribute name="title">
          <string>Stats</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTabWidget" name="statsTabs">
              <property name="styleSheet">
               <string notr="true">QTabWidget::tab-bar {
            alignment: center;
}</string>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="dayTab">
               <attribute name="title">
                <string>Day</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1" colspan="2">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QPushButton" name="prevDay">
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>prev</string>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLabel" name="dayNameLabel">
                      <property name="text">
                       <string>Today</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="nextDay">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>next</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QChartView" name="dayChartView"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="weekTab">
               <attribute name="title">
                <string>Week</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <widget class="QPushButton" name="prevWeek">
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>prev</string>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLabel" name="weekNameLabel">
                      <property name="text">
                       <string>This week</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="nextWeek">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>next</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QChartView" name="weekChartView"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="monthTab">
               <attribute name="title">
                <string>Month</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QPushButton" name="prevMonth">
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>prev</string>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLabel" name="monthNameLabel">
                      <property name="text">
                       <string>November 2023</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="nextMonth">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>100</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>next</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QChartView" name="monthChartView"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
